baselines/exo.md

```diff
@@ -173,29 +195,31 @@ DKIM SHOULD be enabled for any custom domain.
 
 ### Implementation
 
+#### MS.EXO.3.1v1 instructions:
 To enable DKIM, follow the instructions listed on [Steps to Create,
 enable and disable DKIM from Microsoft 365 Defender portal \| Microsoft
-Docs](https://docs.microsoft.com/en-us/microsoft-365/security/office-365-security/use-dkim-to-validate-outbound-email?view=o365-worldwide#steps-to-create-enable-and-disable-dkim-from-microsoft-365-defender-portal).
+Learn](https://learn.microsoft.com/en-us/microsoft-365/security/office-365-security/email-authentication-dkim-configure?view=o365-worldwide#steps-to-create-enable-and-disable-dkim-from-microsoft-365-defender-portal).
+
+1.  Navigate to the **Microsoft 365 Defender portal**.
 
-1.  Navigate to the [Microsoft 365
-    Defender](http://security.microsoft.com) admin center.
+2.  On the left hand side go to **Email & collaboration** > **Policies & Rules**.
 
-2.  Go to Policies & Rules.
+3. From the list of policies, select *Threat Policies**.
```

Comment:
is the *Threat Policies** intentional? or do you meant to use **Threat Policies**